ci/pipeline.yml

```diff
@@ -8,39 +8,75 @@ anchors:
     AWS_ACCESS_KEY_ID: ((aws-access-key-id))
     AWS_SECRET_ACCESS_KEY: ((aws-secret-access-key))
   bosh-creds: &bosh-creds
-    jumpbox_key: |
-      ((jumpbox-key.private_key))
-    ca_cert: |
-      ((ca-cert.certificate))
+    jumpbox_key: ((jumpbox-key))
+    ca_cert: ((ca-cert))
     BOSH_ENVIRONMENT: ((bosh-environment))
 
 resources:
-  - name: kafka-repo
+  - name: boshcli2
+    type: registry-image
+    source:
+      repository: afamuzoka/boshcli2
```

Comment:
Did we not get this into the Smarsh docker repo yet?

Reply:
There is some follow up work we need to do for this - here are the notes from the call I had with Afam and I'll follow up and post this information in [66](https://smarsh.atlassian.net/secure/RapidBoard.jspa?rapidView=460&projectKey=SAAS&modal=detail&selectedIssue=SAAS-66)

- We would need a private docker image which we can't have currently since we are 1 of 1
- We would need a way to automatically maintain pulling down the latest open source image, updating the dockerfile, and pushing it up to smarsh's docker hub (there is an example pipeline for this that Afam showed me I don't have the link) we could clone this pipeline to patch any kind of security issue with the older images.

Those things said - it's possible to just push up the current image into the smarsh docker repo as a public image for the time being and circle back on all of the enhancement discussions.